controllers/front/ProductController.php

```diff
@@ -1215,15 +1215,14 @@ public function getBreadcrumbLinks()
         $categoryDefault = new Category($this->product->id_category_default, $this->context->language->id);
 
         foreach ($categoryDefault->getAllParents() as $category) {
-            if ($category->id_parent != 0 && !$category->is_root_category) {
-                $breadcrumb['links'][] = $this->getCategoryPath($category);
+            if ($category->id_parent != 0 && !$category->is_root_category && $category->active) {
+                $breadcrumb['links'][] = [
+                    'title' => $category->name,
+                    'url' => $this->context->link->getCategoryLink($category),
+                ];
             }
```

Comment:
I would be in favor of getting rid of the `if` statement here (and everywhere it was made like that), fix the one in the method `getCategoryPath()` and return a `array_filter($breadcrumb)` to remove `null` entries. What do you think?